analyzer/src/main/kotlin/managers/Unmanaged.kt

```diff
@@ -102,7 +102,7 @@ class Unmanaged(
 
             else -> {
                 // For all non-GitRepo VCSes derive the name from the VCS URL.
-                val fileUrl = File(vcsInfo.url).takeIf { it.exists() } ?: File(URI(vcsInfo.url))
+                val fileUrl = File(URI(vcsInfo.url).path)
```

Comment:
The variable name `fileUrl` is now IMO misleading, because the value is no more a URL but a path.

Have you considered reverting the original change? (as we know previously it worked except for windows).
I'm not sure, but may  also fixed the issue for windows?

[1] https://github.com/oss-review-toolkit/ort/pull/6407

Reply:
I would like to hear @sschuberth's opinion to learn what was the original issue with Windows. I assume it was related to path separator characters, but are not sure.

Reply:
Yes, it's indeed fine to revert https://github.com/oss-review-toolkit/ort/pull/6402 as its follow-up https://github.com/oss-review-toolkit/ort/pull/6407 also fixes https://github.com/oss-review-toolkit/ort/issues/2900 in a more generic way.